Components/Alert/alert.css

```diff
@@ -0,0 +1,18 @@
+
+.alert{
+    border-radius: 4px;
+    background-color: var(--basic);
+    padding: 0.5rem 1rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 60%;
+}
+
+.alert-link{
+    color: var(--white);
+}
+
+.fa-times{
```

Comment:
I believe you are using fontawsome, in that case, use <i> tag to target

Reply:
If I use `<i>` tag then that will get applied to all other icons in the component library. I want to apply `cursor: pointer` to specific icons only

Reply:
Okay. it's good to go then👍🏻